packages/teleport-project-generator-html/src/plugin-clone-globals.ts

```diff
@@ -48,7 +48,7 @@ class ProjectPluginCloneGlobals implements ProjectPlugin {
               const parsedIndividualFile = load(file.content)
 
               const metaTags = parsedIndividualFile.root().find('meta')
-              parsedEntry('head').prepend(metaTags.length ? metaTags.toString() : metaTagsFromRoot)
+              parsedEntry('head').prepend(metaTags.toString().concat(metaTagsFromRoot))
```

Comment:
Please change with `findAll` something so we get all meta tags instead of the matched first one.